### DISCUSSION TITLE
aws cloudformation deploy command needs --template-url

### DISCUSSION THREAD

**Discussion by success4lyf** [aws/aws-cli#7543] [Q&A] [answered]
aws cloudformation deploy command needs --template-url to be able to deploy our template in a s3bucket with object type as yaml instead of .template

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hey @success4lyf, thanks for reaching out.  If you're requesting a new feature, the easiest way to make sure it is seen is to make a feature request using this [template](https://github.com/aws/aws-cli/issues/new?assignees=&labels=feature-request%2Cneeds-triage&template=feature-request.yml&title=%28short+issue+description%29).  The template will make sure the team has the type and level of detail required to evaluate a potential feature.  Thanks!

**Reply by tim-finnigan** [reply]
(I made an update to fix the link to the template.)
